lib/gax.js

```diff
@@ -95,6 +98,17 @@ exports.CallSettings = CallSettings;
  *   pageToken.
  * @property {boolean=} isBundling - If set to false and the call is configured
  *   for bundling, bundling is not performed.
+ * @example
+ * // suppress bundling for bundled method.
+ * api.bundlingMethod(request, {isBundling: false}, function(err, response) {
```

Comment:
can we make this `api.bundlingMethod(requiredParameter, {optionalParameter: aValue, isBundling: false}, ...`

to clarify that optional parameters and call options are all attributes on the same object?